### DISCUSSION TITLE
Parcel: No preprocessor found for block type i18n

### DISCUSSION THREAD

**Discussion by hpeuscher** [intlify/vue-i18n#831] [Q&A]
First, let me thank you for this great piece of software. Unfortunately, I cannot seem to get <i18n> blocks to work with Parcel, since I always end up with the error message "@parcel/transformer-vue: No preprocessor found for block type i18n".
I found that [parcel supports custom blocks in vue files](https://parceljs.org/languages/vue/#custom-blocks), but how must I proceed to use that for vue-i18n? Any help would be very much appreciated - thanks.

**Comment by kazupon** [comment]
Unfortunately, vue-i18n is not supported percel.  😞 
I need more time to develop as an OSS developer.
I believe in expansion by the community.
